docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -262,7 +259,7 @@
    "id": "000140e1",
    "metadata": {},
    "source": [
-    "and because we are sending a large number of jobs that we would like to execute together, here we use a `Session` to execute all the generated circuits in one block.  Here `args` is the standard SciPy way to supply the additional parameters needed by the cost function."
+    "and because we are sending a large number of jobs that we would like to execute together, here we use a `Session` to execute all the generated circuits in one block. Here `args` is the standard SciPy way to supply the additional parameters needed by the cost function.\n"
```

Comment:
```suggestion
"Because we are sending a large number of jobs that should run together, we use a `session` to execute all the generated circuits in one block. Here `args` is the standard SciPy way to supply the additional parameters needed by the cost function.\n"
```